packages/ui/src/views/MainContent.vue

```diff
@@ -59,6 +59,14 @@
         >
           PREVIEW NFT
         </Button>
+        <a
+          v-if="player.errors.network"
+          @click="addPolygonNetwork()"
+          class="add-polygon"
+        >
+          <svgImage class="metamask" :svg="metamaskIcon" />
+          Polygon Network
```

Comment:
I think this is missing the cursor? Hovering over "Polygon Network" uses the "select text" cursor.